services/raycast/installs.service.js

```diff
@@ -0,0 +1,68 @@
+import Joi from 'joi'
+import { metric } from '../text-formatters.js'
+import { anyInteger } from '../validators.js'
+import { BaseJsonService, NotFound, pathParams } from '../index.js'
+
+const schema = Joi.object({
+  download_count: anyInteger,
+}).required()
+
+export default class RaycastInstalls extends BaseJsonService {
+  static category = 'platform-support'
+
+  static route = {
+    base: 'raycast/installs',
+    pattern: ':user/:extension',
+  }
+
+  static openApi = {
+    '/raycast/installs/{user}/{extension}': {
+      get: {
+        summary: 'Raycast extension downloads count',
+        parameters: pathParams(
+          { name: 'user', example: 'Fatpandac' },
+          { name: 'extension', example: 'bilibili' },
+        ),
+      },
+    },
+  }
+
+  static _cacheLength = 7200
+
+  static render({ user, extension, downloadCount }) {
```

Comment:
Can we use the `renderDownloadsBadge()` helper please. https://github.com/badges/shields/blob/master/services/downloads.js That will apply our standard formatting and things